packages/cli/src/oclif/commands/env/set.js

```diff
@@ -85,7 +94,16 @@ SetEnvCommand.args = {
       'The key-value pairs to set. Keys are case-insensitive. Each pair should be space separated and pairs should be separated by an `=`. For example: `A=123 B=456`',
   }),
 };
-SetEnvCommand.flags = buildFlags();
+SetEnvCommand.flags = buildFlags({
+  commandFlags: {
+    force: Flags.boolean({
+      char: 'f',
+      description:
+        'Force the update of environment variables regardless if the app version is production or not. Use with caution.',
+      hidden: true,
```

Comment:
I think it'd be more clear if `--force` is visible in the help text. Any reason to hide it?

Reply:
You're right; I think in my head I was hoping to discourage folks from modifying live variables by "hiding" the option. But ultimately, they will see how to override via the error shown on `409`. So let's not complicate their lives and just show the option 😆